### PR TITLE
Improve cli comparison tool

### PR DESCRIPTION
**Pull request by bjpirt**
```
Options

  -f, --file string     Specify either the local file path or an S3 URL                               
  -s, --start string    Specify the start timestamp in ISO8601 format                                 
  -e, --end string      Specify the end timestamp in ISO8601 format                                   
  -p, --filter string   Filter based on the last result. Specify either 'failure', 'success', 'both'. 
                        Default is 'failure'                                                          
  -h, --help            Prints this usage guide   
```
Screenshot 2022-07-06 at 11 14 22

### CODE REVIEW

File: src/comparison/cli.ts [hunk 1]
```diff
@@ -1,36 +1,3 @@
...
-  if (!result.xmlParsingMatches) {
-    console.log("XML parsing does not match")
-    console.log(formatXmlDiff(result.debugOutput.xmlParsingDiff))
-  }
-}
-
-if (result.triggersMatch && result.exceptionsMatch && result.xmlOutputMatches && result.xmlParsingMatches) {
-  console.log("Comparison matches")
-}
+main().then(() => {})
```
Comment (daviesjamie) [inline src/comparison/cli.ts:3]: Do you need the `.then` chained on here?

**Review by daviesjamie — COMMENTED** [review]
Could we add a script to the `package.json`? So instead of 

```
npx ts-node src/comparison/compare.ts
```

We could run something like

```
npm run compare
```

File: src/comparison/cli.ts (hunk 1 above)
Comment (bjpirt) [reply src/comparison/cli.ts:3]: I thought we did because it's an async function but it runs fine without so I'll remove

**Comment by bjpirt** [comment]
> Could we add a script to the `package.json`? So instead of
> 
> ```
> npx ts-node src/comparison/compare.ts
> ```
> 
> We could run something like
> 
> ```
> npm run compare
> ```

There's already a `compare` command but I think this one should probably replace it so I'll update

**Review by daviesjamie — DISMISSED** [review]

**Review by daviesjamie — APPROVED** [review]

**Pull request merged**